packages/connect-express/package.json

```diff
@@ -41,7 +41,7 @@
   },
   "peerDependencies": {
     "express": "^4.18.2 || ^5.0.1",
-    "@bufbuild/protobuf": "^2.2.0",
+    "@bufbuild/protobuf": "^2.2.3",
```

Comment:
Does `@bufbuild/protobuf` v2.2.3 include any changes we need for `@connectrpc/connect-express`? If not, wouldn't it be better to keep the lowest compatible version at v2.2.0?

Same for the other packages with constraints relevant for end-users.